packages/astro/src/vite-plugin-astro/index.ts

```diff
@@ -82,7 +87,7 @@ export default function astro({ config, devServer }: AstroPluginOptions): vite.P
         });
 
         // macOS fix: remove null chars generated by compiler
-        if (os.platform() === 'darwin') {
+        if (platform === 'darwin') {
```

Comment:
This hack should be removed, but at least now we’re not performing a check on every file load (dunno how bad that is but I frequently find Vite caching `os.platform()` so I figured why not)